README.md

```diff
@@ -9,6 +9,25 @@ This module provisions by default a set of basic AWS Config Rules. In order to a
 aws_config_rules = ["ACCESS_KEYS_ROTATED", "ALB_WAF_ENABLED"]
 ```
 
+## Datadog Integration
+
+This module supports an optional Datadog-AWS integration. This integration makes it easier for you to forward metrics and logs from your AWS account to Datadog.
+
+In order to enable the integration, you can pass an object to the variable `datadog` containing the following attributes:
+- `api_key`: sets the Datadog API key
+- `enable_integration`: set to `true` to configure the [Datadog AWS integration](https://docs.datadoghq.com/integrations/amazon_web_services/)
+- `install_log_forwarder`: set to `true` to install the [Datadog Forwarder](https://docs.datadoghq.com/serverless/forwarder/)
```

Comment:
```suggestion
In order to enable the integration, you can pass an object to the variable `datadog` containing the following attributes:

- `api_key`: sets the Datadog API key
- `enable_integration`: set to `true` to configure the [Datadog AWS integration](https://docs.datadoghq.com/integrations/amazon_web_services/)
- `install_log_forwarder`: set to `true` to install the [Datadog Forwarder](https://docs.datadoghq.com/serverless/forwarder/)
```

According to Markdown style guide there should be a separate line between groups of bullets. Nitpicking a bit but if we add a linter it'll highlight this.